ios/RCTTwilio/RCTTwilio.m

```diff
@@ -31,7 +31,12 @@ @implementation RCTTwilio {
 }
 
 RCT_EXPORT_METHOD(initWithToken:(NSString *) token) {
-    _phone = [[TCDevice alloc] initWithCapabilityToken:token delegate:self];
+    if (_phone == nil) {
+      _phone = [[TCDevice alloc] initWithCapabilityToken:token delegate:self];
+    } else {
+      [_phone updateCapabilityToken:token];
```

Comment:
Should this be it's own exported method?
Then you can call either init or update from the JS, otherwise this method is a bit of hidden magic for updating the token.
@ream88 How are you using this method in your code? Are you calling `initWithToken` more than once?